big-practice/src/js/dashboard.js

```diff
@@ -1,10 +1,31 @@
+import isValidForm from './validation';
+
 // Query elements
 const tableBody = document.getElementById('tableBody');
+const nameMovieInput = document.getElementById('nameMovie');
+const directorInput = document.getElementById('director');
+const nationInput = document.getElementById('nation');
+const createBtn = document.getElementById('createBtn');
+const cancelBtn = document.getElementById('cancelBtn');
 const accountName = document.querySelector('.account-name');
 const addBtn = document.querySelector('.add-btn');
 const modal = document.querySelector('.modal');
 const moviesApi = 'http://localhost:3000/movies';
 
+// Messages
+const MESSAGES = {
```

Comment:
Could you group all MESSAGE, Configuration value (ex: api) into one js file like Validation?